docs/UserGuide.md

```diff
@@ -744,9 +770,6 @@ Command          | Format
 #### Alphanumeric
 Consists of only letters and numbers.
 
-#### Asset
-An item or amenity of logistical significance.
-
```

Comment:
Perhaps we should keep this? To be entirely unambiguous as even in the industry, logistics managers might use the term differently or use a different term such as `wares`.